x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/ErrorsOverviewLink.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import styled from 'styled-components';
+import { idx } from 'x-pack/plugins/apm/common/idx';
+import { KibanaLink } from 'x-pack/plugins/apm/public/components/shared/Links/KibanaLink';
+import { legacyEncodeURIComponent } from 'x-pack/plugins/apm/public/components/shared/Links/url_helpers';
+import { Transaction } from '../../../../../typings/es_schemas/Transaction';
+import { colors, fontSizes } from '../../../../style/variables';
+
+const LinkLabel = styled.span`
+  font-size: ${fontSizes.small};
```

Comment:
It's a little on the small side, we should use `fontSize` instead so it renders as 14px.